### ISSUE TITLE
replace ASCII weather icons to something else

### ISSUE THREAD

**Issue by lkthomas** [chubin/wttr.in#130]
is it possible to replace icons? I want to make it look nice on browser.

**Comment by chubin** [comment]
@lkthomas Do you mean, you want to use some other icons? Graphical icons? But the layout and everything else stays the same? So you want to change the icons only?
Do you have some examples of icons you want to use?

**Comment by lkthomas** [comment]
http://www.freepik.com/free-vector/hand-drawn-collection-of-clouds-in-different-weather-conditions_1000956.htm#term=weather&page=1&position=30

I just random Google on "weather icons", I think icon section should be able to change, but I think wttr.in program should be able to resize icons to a size which wouldn't mess up the format at all, eventually I want to display it on browser, not terminal.

**Comment by book** [comment]
Unicode has a bunch of weather symbols, which could be used for terminal output:

* 🌣 - U+1F323 - WHITE SUN
* 🌤 - U+1F324 - WHITE SUN WITH SMALL CLOUD
* 🌥 - U+1F325 - WHITE SUN BEHIND CLOUD
* 🌦 - U+1F326 - WHITE SUN BEHIND CLOUD WITH RAIN
* 🌧 - U+1F327 - CLOUD WITH RAIN
* 🌨 - U+1F328 - CLOUD WITH SNOW
* 🌩 - U+1F329 - CLOUD WITH LIGHTNING
* 🌪 - U+1F32A - CLOUD WITH TORNADO
* ☀ - U+02600 - BLACK SUN WITH RAYS
* ☁ - U+02601 - CLOUD
* ☂ - U+02602 - UMBRELLA
* ☇ - U+02607 - LIGHTNING
* ☈ - U+02608 - THUNDERSTORM
* ☔ - U+02614 - UMBRELLA WITH RAIN DROPS
* ☼ - U+0263C - WHITE SUN WITH RAYS
* ⛆ - U+026C6 - RAIN
* ⛈ - U+026C8 - THUNDER CLOUD AND RAIN

**Comment by chubin** [comment]
Yes, you are right, thi symbols are used in the one-line version (not yet released)

**Comment by lkthomas** [comment]
Does unicode icon could expand to bigger icon? the size is too small.

**Comment by chubin** [comment]
In browser yes, in terminal no

**Comment by lkthomas** [comment]
well eventually I want to show it on browser, it will be great if we could adjust icon size
